pyflagser/__init__.py

```diff
@@ -1,17 +1,25 @@
 from ._version import __version__
 
-from .flagio import load_unweighted_flag, load_weighted_flag, \
-    save_unweighted_flag, save_weighted_flag
-from .flagser import flagser_unweighted, flagser_weighted
-from .flagser_count import flagser_count_unweighted, \
-    flagser_count_weighted
+__all__ = ['__version__']
 
-__all__ = ['load_unweighted_flag',
-           'load_weighted_flag',
-           'save_unweighted_flag',
-           'save_weighted_flag',
-           'flagser_unweighted',
-           'flagser_weighted',
-           'flagser_count_unweighted',
-           'flagser_count_weighted',
-           '__version__']
+# Allows to have access to the version information without
+# installating package + dependencies
```

Comment:
What is this needed for?

Reply:
>  This way, people can build the docs even if they don't have the developer install (similarly to how we did a lot of work to make the tests work with the wheels).

I copy/paste your statement, that I find really clear. This is from a discussion in this PR from 11 June 2021

Reply:
Cool, yes!